src/library/libraryfeature.cpp

```diff
@@ -20,7 +20,8 @@ LibraryFeature::~LibraryFeature() {
     
 }
 
-QString LibraryFeature::getPlaylistFile() {    
+QStringList LibraryFeature::getPlaylistFiles(QFileDialog::FileMode mode)
+{
```

Comment:
Tiny nit: please put the opening brace on the same line as the function signature